linter/allowlists/default.txt

```diff
@@ -52,6 +53,7 @@ Cross
 CSS
 CSV
 Dalley
+Debug
```

Comment:
This should go to the German or English allowlist.

Reply:
Thank you!